### ISSUE TITLE
Does this work on IOS Simulator?

### ISSUE THREAD

**Issue by balaji830** [MicrosoftDocs/appcenter-docs#744]
I created an example app with Xamarin Forms. AppCenter Auth works fine with Identity providers (Both MS and Google) I tested on Android. Android documentation was adequate to work with. 
But on IOS Simulator, it always comes back with an exception "User sign-in failed. Internet Connection is down". Has anyone got this one to work on IOS? I am wondering if this documentation is adequate or am I missing something?
 **Update on Error:**
 **AppCenter.Start("65d*****-******", typeof(Auth)) line of code raise an exception in IOS. Does work in Android. As per documentation, I have modified the info.plist and added the Entitlements.plist and still get the error "Failed to start Auth service: Custom URL Scheme for Auth not found". Is there something else in the documentation that is missing?**

I understand the App Center Auth is still in preview. I am trying to see if this makes sense for me to recommend it to my customer (they are an Azure customer and wants to stay in this environment if possible). If I cannot provide a POC to show them on a simulator. It is a hard sell.

**Comment by balaji830** [comment]
AppCenter.Start("65d*****-******", typeof(Auth)) line of code raise an exception in IOS. Does work in Android. As per documentation, I have modified the info.plist and added the Entitlements.plist and still get the error "Failed to start Auth service: Custom URL Scheme for Auth not found". Is there something else in the documentation that is missing?

**Comment by amchew** [comment]
Hey @balaji830, thanks for reaching out - we'll investigate and get back!

**Comment by balaji830** [comment]
IOS settings documentation (Step 4, IOS Additional settings section) may be wrong (pertained to Xamarin Form). The documentation says to add msal{appsecret} to info.plist but that alone does not work. I got it working by adding msal{appsecret}://auth in info.plist. in addition to what is provided in documentation. I created multiple test projects and followed the steps. This addition that I have listed is necessary for it to work. 
Couple of more questions
1. Is App center Auth supported on UWP?
2. The decode token code provided works on Android but **does not work on IOS**.
